client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/notices/notices.scss

```diff
@@ -0,0 +1,11 @@
+.pattern-assembler {
+	.components-snackbar-list {
+		overflow: hidden;
+		bottom: 8px;
+		padding-inline-end: 8px;
```

Comment:
This makes the notices LTR-ready, showing them on the left.

<img width="341" alt="Screenshot 2566-03-23 at 12 57 12" src="https://user-images.githubusercontent.com/1881481/227116517-68ff2193-51d9-4e7c-8828-0b4a2cc03553.png">

Note: On the screenshot, the texts are in English because the translations are still not ready.

Reply:
Thanks, better 16px 👍 as in the editor.